test/sales_engine_test.rb

```diff
@@ -82,7 +81,7 @@ def test_it_has_a_default_item_location
   end
 
   def test_it_passes_parsed_item_data_to_item_repository
-    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv"}
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.